StripePaymentsUI/StripePaymentsUI/Source/UI Components/PaymentMethodMessagingView.swift

```diff
@@ -170,47 +170,73 @@ import WebKit
 // MARK: - Helpers
 
 extension PaymentMethodMessagingView {
-    static func makeAttributedString(
-        from html: String,
-        font: UIFont
-    ) async throws -> NSAttributedString {
+    /// A regex that matches <img> tags and captures its url
+    static let imgTagRegex = try! NSRegularExpression(pattern: "<img src=\"(.*?)\">")
+    
+    /// - Returns: The given `html` string with <img> tags replaced with <a href> tags, and a list of the URLs contained in the replaced tags
+    static func HTMLReplacingImageTags(html: String) -> (String, [URL]) {
+        var html_with_img_tags_replaced = html
+        var images = [URL]()
+        for match in imgTagRegex.matches(in: html, range: NSRange(html.startIndex..., in: html)).reversed() {
+            guard
+                let rangeOfImgTag = Range(match.range, in: html),
+                let rangeOfURL = Range(match.range(at: 1), in: html)
+            else {
+                continue
+            }
+            let url = html[rangeOfURL]
+            html_with_img_tags_replaced.replaceSubrange(rangeOfImgTag, with: "<a href=\"\(url)\">_</a>")
+            // Fetch the image
+            guard let URL = URL(string: String(url)) else { continue }
+            images.append(URL)
+        }
+        return (html_with_img_tags_replaced, images)
+    }
+    
+    static func makeCSS(for font: UIFont) -> String {
```

Comment:
nit: 

```suggestion
    static func makeCss(for font: UIFont) -> String {
```

Reply:
I don't feel strongly but I _think_ this is matches https://www.swift.org/documentation/api-design-guidelines/ (under Follow case conventions)

Reply:
Ah, yes you are right.